.github/workflows/main.yml

```
@@ -18,5 +18,7 @@ jobs:
       - uses: actions/setup-node@v1
```

actionlint (1.7.4)

the runner of "actions/setup-node@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)